docs/development.md

```diff
@@ -172,6 +173,14 @@ Several configurations are provided by default:
 * `Local tests` - Runs `src/test/testng-local.xml`, all the tests that need to be run locally by developers. `Dev green` means passing all the tests in this configuration.
 * `Failed tests` - Runs `test-output/testng-failed.xml`, which is generated if a test run results in some failures. This will run only the failed tests.
 
+#### GodMode
```

Comment:
Wrote an alternative explanation. Use all/part of it as you see fit, as is or in a revised form.

----
To ensure that the UI generated by the app is same as the expected UI, some of our tests compare the raw html text of Web pages generated by the app (let's call them _actual-html_) against the corresponding content of _expected-html_ files previously recorded. This means these _expected-html_ files need to be updated every time you do a UI-related code change (e.g. add a new feature to the UI). Assuming you have already done the UI-related code change and tested in manually, the procedure to update expected-html files of a test is as follows:
1. Run the test in a special mode we call the _God Mode_ (see [godmode.md](godmode.md) for more info) which captures the Web pages being generated by the app and use that new content to overwrites existing expected-html pages.
1. Manually examine the diff of the updated expected-html files to ensure the changes are as expected.
After that, future runs of the test (in normal mode) will be comparing the actual-html against these updated expected-html files that you generated using God Mode. 

Note that the same God Mode technique is used to ensure the content of emails generated by the app are as expected.